package.json

```diff
@@ -114,5 +114,8 @@
     "testMatch": [
       "**/*.test.js"
     ]
+  },
+  "dependencies": {
+    "sane": "^2.0.0"
```

Comment:
Shouldn't this be https://github.com/facebook/jest/blob/master/packages/jest-haste-map/package.json#L15?